templates/openstack/what-is-openstack.html

```diff
@@ -2,206 +2,601 @@
+      <h3 class="p-card__title p-heading--three">Neutron</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Neutron provides network connectivity between OpenStack instances, enabling multi-VM deployments. For this purpose, Neutron uses various software defined networking (SDN) technologies, including Open Virtual Network (OVN), Open vSwitch (OVS), Juniper Contrail, Cisco ACI, etc. </p>
+    </article>
+  </div>
+  <div class="row u-equal-height">
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Cinder</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Cinder is a storage component that is responsible for provisioning, management and termination of persistent block devices. Those can be later attached to the instances running on OpenStack to enable persistent block storage.</p>
+    </article>
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Swift</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Swift is another storage component that provides a highly available and scalable object storage service similar to Amazon S3. It enables storing and retrieving unstructured data objects using a RESTful API for both OpenStack services and instances running on the cloud.</p>
+    </article>
+    <article class="col-4 p-card">
+      <h3 class="p-card__title p-heading--three">Keystone</h3>
+      <hr class="u-sv1" />
+      <p class="p-card__content">Keystone serves as an identity service, providing authentication and authorization functions for the users in order to enable multi-tenancy. Keystone can be easily integrated with external identity systems, such as lightweight directory access protocol (LDAP) or Active Directory.</p>
+    </article>
+  </div>
+</section>
+
 <section class="p-strip">
   <div class="row">
     <div class="col-8">
       <h2>Companies involved in OpenStack</h2>
-      <p>OpenStack was originally launched in July 2010 by Rackspace and NASA as an open source initiative that combined NASA's Nebula platform and Rackspace's Cloud Files platform.</p>
+      <p>OpenStack was originally launched in July 2010 by Rackspace and NASA as an open source initiative that combined NASA’s Nebula platform and Rackspace’s Cloud Files platform.</p>
       <p>Today, hundreds of companies contribute to OpenStack code, with many more playing a part in the broader community.</p>
     </div>
   </div>
-</section>
-
-<section class="p-strip u-no-padding--top u-no-padding--bottom">
   <div class="u-fixed-width">
-    <h3 class="p-muted-heading">Influential contributors to openStack</h3>
-  </div>
-  <ul class="row">
-    <li class="col-2 col-small-2 col-medium-2">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/403148e0-2018-logo-canonical.svg",
-          alt="Canonical",
-          height="144",
-          width="144",
+    <ul class="row">
+      <li class="col-2 col-small-2 col-medium-2">
```

Comment:
There is a logo pattern you can use on Vanilla https://vanillaframework.io/docs/patterns/logo-section